config/motoros2_config.yaml

```diff
@@ -177,9 +177,11 @@ namespace_tf: true
 # When using a 7 axis robot arm with an elbow joint (E) in the middle of the
 # arm, the elbow axis should be listed last (SLURBTE).
 #
-# DEFAULT: "group_x/joint_y"
+# DEFAULT:
+# - Single-group: "joint_y"
+# - Multi-group: "group_x/joint_y"
```

Comment:
```suggestion
# - Single-group: "joint_n"
# - Multi-group: "group_n/joint_m"
```

let's switch to more common notation for this sort of thing. It would also align better with the FAQ entry.

Reply:
Good point